orchestrator/dags/general_data_import.py

```diff
@@ -89,15 +89,15 @@ def extract_timetable_lines(ti):
                 continue    
 
 
-def convert_to_time(str_hour: str):
+def convert_to_time(str_hour: str) -> datetime.date:
     if '24.' not in str_hour:
         hour = datetime.strptime(str_hour, '%H.%M').time()
     else:
         hour = datetime.strptime(str_hour.replace('24.', '00.'), '%H.%M').time()
     return hour
 
 
-def extract_timetable():
+def extract_timetable() -> list:
```

Comment:
In this case it's better to use `List` imported from `typing` module, because we can specify the content of the list. 
If we were using newer Python, at least 3.9, then we could specify content as well for built-in `list` type.
Take a look at this explanation: https://stackoverflow.com/a/39458225/1937393

```suggestion
def extract_timetable() -> List[TimeTable]:
```